src/main.ts

```diff
@@ -1,114 +1,59 @@
 import * as fs from 'fs'
 import * as cp from 'child_process'
 import * as core from '@actions/core'
-import * as toolCache from '@actions/tool-cache'
-import path from 'path'
+import * as path from 'path'
 
 async function run(): Promise<void> {
   try {
+    const localAppDataPath = process.env.LocalAppData;
     // get version number from input
     const version = core.getInput('version')
 
-    // get architecture and os
-    const architecture = process.arch
+    // get os
     const os = process.platform
 
-    // map for different platform and arch
-    const extensionMap = {
-      linux: '.tar.gz',
-      darwin: '.zip',
-      win32: '.zip'
+    let windowsInstallScript = `powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression"`
+    let linuxOrMacOSInstallScript = `sudo curl -fsSL https://aka.ms/install-azd.sh | bash`
```

Comment:
sudo is required running the bash script, not for calling curl

```suggestion
    let linuxOrMacOSInstallScript = `curl -fsSL https://aka.ms/install-azd.sh | sudo bash`
```